guides/blending/client-side-blending/README.md

```diff
@@ -0,0 +1,146 @@
+
+## Composing Layouts
+
+With the MedicalProvider and PetList example we have two layouts that we would like to blend:
+
+<div class="code-name">PetList layout</div>
+
+```html 
+<style>
+    @import '/PetList/style.css';
+</style>
+<div class="pet-list-wrapper">
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-name"></slot>
+        <slot name="petlist/details-age-and-animal"></slot>
+    </div>
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-owner-name"></slot>
+        <slot name="petlist/details-weight"></slot>
+    </div>
+    <slot name="petlist/details-list-link"></slot>
+</div>
+```
+
+<div class="code-name">MedicalProvider layout</div>
+
+```html
+<slot name="medicalrecord/records-list-headline"></slot>
+<slot name="medicalrecord/records-list-table"></slot>
+```
+
+When these two layouts are merged with server-side blending, the resulting layout looks like this:
+
+```html
+<style>
+    @import '/PetList/style.css';
+</style>
+<div class="pet-list-wrapper">
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-name"></slot>
+        <slot name="petlist/details-age-and-animal"></slot>
+    </div>
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-owner-name"></slot>
+        <slot name="petlist/details-weight"></slot>
+    </div>
+    <slot name="petlist/details-list-link"></slot>
+</div>
+<slot name="medicalrecord/records-list-headline"></slot>
+<slot name="medicalrecord/records-list-table"></slot>
+```
+
+The layout from MedicalRecord is appended at the end of the PetList wrapper. To blend it, we will move the MedicalRecord table and headline into the `div class="pet-list-wrapper"` and expand the width of the wrapper to fit the table:
+
+```html
+<style>
+    @import '/PetList/style.css';
+    .pet-list-wrapper {
+        max-width: 750px;
+    }
+</style>
+<div class="pet-list-wrapper">
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-name"></slot>
+        <slot name="petlist/details-age-and-animal"></slot>
+    </div>
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-owner-name"></slot>
+        <slot name="petlist/details-weight"></slot>
+    </div>
+    <slot name="medicalrecord/records-list-headline"></slot>
+    <slot name="medicalrecord/records-list-table"></slot>
+    <slot name="petlist/details-list-link"></slot>
+</div>
+```
+
+We have now produced the result shown in the image above; the view from the MedicalRecord app has been neatly integrated with the view from PetList.
+
+## Creating Layouts
+
+The tool for creating these layouts is the [CompositionEditor](https://github.com/starcounterapps/CompositionEditor). It's a Starcounter app that can be opened att any page with `Ctrl + E`. There are two main concepts in the CompositionEditor - the identifier and layout. 
```

Comment:
I'd add a note that CompositionEditor is more-or-less just a HTML editor, that allows you to edit the Shadow DOM HTML layout composition.

Then, hopefully the reader would catch it easier, what it is and what he/she can do with it.